### ISSUE TITLE
error: redefinition of ‘struct rtentry’  struct rtentry

### ISSUE THREAD

**Issue by lindsaymacvean** [openweave/openweave-core#2]
I tried running 
`make -f Makefile-Standalone`
and it ran for a while and then ended with this error
```
Making all in warm
  CXX      ../../src/warm/libWarm_a-WarmCore.o
In file included from ../../src/include/InetLayer/TunEndPoint.h:59:0,
                 from ../../src/include/InetLayer/InetLayer.h:85,
                 from ../../src/include/BleLayer/BleLayer.h:67,
                 from ../../src/include/Weave/Core/WeaveCore.h:34,
                 from ../../src/include/Warm/Warm.h:80,
                 from /home/lmacvean/workarea/openweave-core/src/warm/../../src/warm/WarmCore.cpp:26:
/usr/include/linux/route.h:31:8: error: redefinition of ‘struct rtentry’
 struct rtentry {
        ^~~~~~~
In file included from ../../src/include/InetLayer/TunEndPoint.h:52:0,
                 from ../../src/include/InetLayer/InetLayer.h:85,
                 from ../../src/include/BleLayer/BleLayer.h:67,
                 from ../../src/include/Weave/Core/WeaveCore.h:34,
                 from ../../src/include/Warm/Warm.h:80,
                 from /home/lmacvean/workarea/openweave-core/src/warm/../../src/warm/WarmCore.cpp:26:
/usr/include/net/route.h:31:8: note: previous definition of ‘struct rtentry’
 struct rtentry
        ^~~~~~~
In file included from ../../src/include/InetLayer/TunEndPoint.h:63:0,
                 from ../../src/include/InetLayer/InetLayer.h:85,
                 from ../../src/include/BleLayer/BleLayer.h:67,
                 from ../../src/include/Weave/Core/WeaveCore.h:34,
                 from ../../src/include/Warm/Warm.h:80,
                 from /home/lmacvean/workarea/openweave-core/src/warm/../../src/warm/WarmCore.cpp:26:
/usr/include/linux/ipv6_route.h:43:8: error: redefinition of ‘struct in6_rtmsg’
 struct in6_rtmsg {
        ^~~~~~~~~
In file included from ../../src/include/InetLayer/TunEndPoint.h:52:0,
                 from ../../src/include/InetLayer/InetLayer.h:85,
                 from ../../src/include/BleLayer/BleLayer.h:67,
                 from ../../src/include/Weave/Core/WeaveCore.h:34,
                 from ../../src/include/Warm/Warm.h:80,
                 from /home/lmacvean/workarea/openweave-core/src/warm/../../src/warm/WarmCore.cpp:26:
/usr/include/net/route.h:57:8: note: previous definition of ‘struct in6_rtmsg’
 struct in6_rtmsg
        ^~~~~~~~~
make[4]: *** [Makefile:622: ../../src/warm/libWarm_a-WarmCore.o] Error 1
make[3]: *** [Makefile:578: all-recursive] Error 1
make[2]: *** [Makefile:655: all-recursive] Error 1
make[1]: *** [Makefile:580: all] Error 2
make: *** [Makefile-Standalone:380: do-build-x86_64-unknown-linux-gnu] Error 2
```

**Comment by Vyrastas** [comment]
Hi @lindsaymacvean, what Linux distribution do you have?

**Comment by robszewczyk** [comment]
Fixed in commit 7d36c2d503ab3a61342a2b2893fff0dd4d779e7c
